styles/_containerQueries.scss

```diff
@@ -0,0 +1 @@
+$cq1024: 1024px;
```

Comment:
i think we forgot to circle back on trying to get this container query variable to work, do we want to remove this for now?

Reply:
Oh good call...I'll delete the file and add a "v2" ticket for that in case we have time for stuff like that.